apps/chat/src/store/prompts/prompts.epics.ts

```diff
@@ -724,31 +723,31 @@ const deleteChosenPromptsEpic: AppEpic = (action$, state$) =>
       const chosenFolderIds = PromptsSelectors.selectChosenFolderIds(
         state$.value,
       );
-      const searchTerm = PromptsSelectors.selectSearchTerm(state$.value);
-      const prompts = PromptsSelectors.selectPrompts(state$.value);
-      const promptIds = prompts.map((prompt) => prompt.id);
+      const allPromptIds = PromptsSelectors.selectPrompts(state$.value).map(
+        (prompt) => prompt.id,
+      );
+      const searchedPromptIds = PromptsSelectors.selectSearchedPrompts(
+        state$.value,
+      ).map((prompt) => prompt.id);
       const folders = PromptsSelectors.selectFolders(state$.value);
       const deletedPromptIds = uniq([
-        ...chosenPromptIds.filter((id) =>
-          doesEntityContainSearchTerm(
-            parsePromptApiKey(id) as unknown as ShareEntity,
-            searchTerm,
-          ),
-        ),
-        ...promptIds.filter(
-          (id) =>
-            chosenFolderIds.some((folderId) => id.startsWith(folderId)) &&
-            doesEntityContainSearchTerm(
-              parsePromptApiKey(id) as unknown as ShareEntity,
-              searchTerm,
-            ),
+        ...chosenPromptIds,
+        ...searchedPromptIds.filter((id) =>
+          chosenFolderIds.some((folderId) => id.startsWith(folderId)),
```

Comment:
I think we just need to replace `doesEntityContainSearchTerm` by `searchedPromptIds.includes(id)` and don't change other parts
```suggestion
       ...chosenPromptIds.filter((id) =>
          searchedPromptIds.includes(id),
        ),
        ...promptIds.filter(
          (id) =>
            chosenFolderIds.some((folderId) => id.startsWith(folderId)) &&
            searchedPromptIds.includes(id),
        ),
      ]);
      const filteredPrompts = prompts.filter(
        (prompt) => !deletedPromptIds.includes(prompt.id),
      );
```
+ `if (promptIds.length) {` -> `if (searchedPromptIds.length) {`